src/lib/simulations/orchestrator/runCustomBenchmarkOrchestrator.ts

```diff
@@ -39,7 +39,7 @@ export async function runCustomBenchmarkOrchestrator(benchmarkForm: BenchmarkFor
   return orchestrator
 }
 
-function generateSimulationMetadata(benchmarkForm: BenchmarkForm) {
+function generateSimulationMetadata(benchmarkForm: Pick<BenchmarkForm, 'characterId' | 'teammate0' | 'teammate1' | 'teammate2' | 'subDps'>) {
```

Comment:
I prefer avoiding pick when possible, these should just be solid BenchmarkForms, it was just missing setConditionals being set when I wrote the test helper